src/query/parser.ts

```diff
@@ -19,20 +19,21 @@ export class ParsingError extends Error {
   }
 }
 
-export function parseQuery(raw: string): Query {
+export function parseQuery(pendingQuery: any): Query {
```

Comment:
Can we use `PendingQuery` as the type here? You'll need to export the type from `injector.ts`